packages/elements/src/internals/utils/inspector/browser/index.ts

```diff
@@ -0,0 +1,20 @@
+import { isTruthy } from '@clerk/shared/underscore';
+// @ts-expect-error - Exists only in devDependencies; Removed by tsup for production builds
+// eslint-disable-next-line import/no-unresolved
+import { createBrowserInspector } from '@statelyai/inspect';
+
+export const getInspector = () => {
+  if (
+    typeof window !== 'undefined' &&
+    process.env.NODE_ENV === 'development' &&
+    isTruthy(process.env.NEXT_PUBLIC_CLERK_ELEMENTS_DEBUG_UI)
```

Comment:
In the `console` file you're also checking for the non-Next.js version, we should do this here, too.
So `CLERK_ELEMENTS_DEBUG_UI`